### DISCUSSION TITLE
problem with my own data

### DISCUSSION THREAD

**Discussion by kratzera** [egeulgen/pathfindR#163] [Q&A] [answered]
Hej there,
I am very new to performing my own analysis in Rstudio. So, I saw your script and it works with your test data, but when I upload my on data it gives me the following error:
Error in input_testing(input, p_val_threshold) : 
  p values must all be numeric
My data is the following:
spc_tbl_ [1,204 × 3] (S3: spec_tbl_df/tbl_df/tbl/data.frame)
 $ Gene.symbol: chr [1:1204] "AAK1" "ABCA13" "ABCA5" "ABCB9" ...
 $ logFC      : num [1:1204] 0.739 0.72 0.771 0.717 0.804 -0.788 1.05 -1.29 0.775 1.83 ...
 $ adj.P.Val  : num [1:1204] 2.68e-04 1.84e-02 1.94e-02 6.69e-03 2.56e-05 4.20e-04 3.56e-02 5.93e-03 4.28e-02 5.68e-08 ...
 - attr(*, "spec")=
  .. cols(
  ..   Gene.symbol = col_character(),
  ..   logFC = col_number(),
  ..   adj.P.Val = col_number()
  .. )
 - attr(*, "problems")=<externalptr> 

Can you please advise me on this. It would be very helpful if I can make this work.
Thanks, H

**Comment by egeulgen** [comment, marked as answer]
Please convert your data into a data frame via as.data.frame() and this should resolve the issue

**Reply by kratzera** [reply]
Thank you! It worked. I am very excited.
